CITATION.cff

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+title: "EModelRunner"
+doi: 10.5281/zenodo.8116075
+url: https://doi.org/10.5281/zenodo.8116075
+abstract: "EModelRunner is a python library designed to run the cell models provided by the Blue Brain portals in a simple and straightforward way. It consists of two main modules. The first module is designed to show the synapse plasticity phenomenonand to run the cells and reproduce the results from the paper 'A calcium-based plasticity model for predicting long-term potentiation and depression in the neocortex'. The second module is designed to run the cells from the Blue Brain Project SomatoSensoryCortex portal. Note that EModelRunner is a convenience tool and does not aim to be a universal simulator."
```

Comment:
There is also the 3rd module (the thalamus) which is added in a later version. I don't think we have to mention the number of modules (since in the future new ones can be added). 


E.g. this can be a better alternative
"EModelRunner is a Python package designed to run single cell models from public Blue Brain Project resources. This includes both portals like the Somatosensory Cortex portal or the Thalamus microstudio, and research findings such as those from 'A calcium-based plasticity model for predicting long-term potentiation and depression in the neocortex'. It offers an accessible way to explore these diverse models and findings, all in one convenient package. This tool, while focused on ease of use, is not a universal simulator but rather a convenient way to access and use specific cell models from Blue Brain's resources."

Reply:
I have also updated the .zenodo file with that description